src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -25,6 +25,14 @@
 
     private boolean distributedSlack = true;
 
+    private BalanceType balanceType = BalanceType.PROPORTIONAL_TO_GENERATION_P; // Default slack distribution on generators.
+
+    public enum BalanceType {
+        PROPORTIONAL_TO_GENERATION_P,
+        // PROPORTIONAL_TO_GENERATION_P_MAX, Not implemented yet.
```

Comment:
This is the one that has been implemented. PROPORTIONAL_TO_GENERATION_P has not been done.

Reply:
Yes, you are right.